src/ecalc/libraries/libecalc/common/libecalc/input/yaml_types/components/generator_set.py

```diff
@@ -0,0 +1,50 @@
+from typing import Any, Dict, List, Type
+
+from libecalc.dto.base import ConsumerUserDefinedCategoryType
+from libecalc.input.yaml_types import YamlBase
+from libecalc.input.yaml_types.components.category import CategoryField
+from libecalc.input.yaml_types.components.legacy.yaml_electricity_consumer import (
+    YamlElectricityConsumer,
+)
+from libecalc.input.yaml_types.placeholder_type import PlaceholderType
+from libecalc.input.yaml_types.schema_helpers import (
+    replace_temporal_placeholder_property_with_legacy_ref,
+)
+from libecalc.input.yaml_types.temporal_model import TemporalModel
```

Comment:
should we be consistent and rename _all_ yaml types to have **Yaml** as a prefix? WIP perhaps?

Reply:
wip yes